py3status/modules/online_status.py

```diff
@@ -21,13 +20,7 @@
 @author obb
 """
 
-import os
-import subprocess
-try:
-    # python3
-    from urllib.request import urlopen
-except:
-    from urllib2 import urlopen
+from urllib.parse import urlsplit
```

Comment:
`extra/python2-requests` already installed. After that setup command, I tried few modules and kept getting: `Error: unknown command`.

Also, I think I'm nearly out of modules to optimize too. 😓